dataprofiler/profilers/profiler_options.py

```diff
@@ -1583,6 +1587,27 @@ def _numeric_stats_disabled_presets(self) -> None:
         self.set({"*.float.is_numeric_stats_enabled": False})
         self.set({"structured_options.text.is_numeric_stats_enabled": False})
 
+    def _memory_optimization_presets(self) -> None:  # split into more dicts
```

Comment:
i don't think this comment is needed.... 

```suggestion
    def _memory_optimization_presets(self) -> None: 
```